deepchecks/vision/checks/distribution/whole_dataset_drift.py

```diff
@@ -0,0 +1,131 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Module contains the domain classifier drift check."""
+from collections import OrderedDict
+from typing import Any, List
+
+from deepchecks.core import CheckResult, DatasetKind
+from deepchecks.vision import Context, TrainTestCheck
+from deepchecks.core.check_utils.whole_dataset_drift_utils import run_whole_dataset_drift
+import pandas as pd
+
+__all__ = ['WholeDatasetDrift']
+
+DEFAULT_IMAGE_PROPERTIES = ['aspect_ratio',
+                            'area',
+                            'brightness',
+                            'normalized_red_mean',
+                            'normalized_green_mean',
+                            'normalized_blue_mean']
+
+
+class WholeDatasetDrift(TrainTestCheck):
+    """
+    Calculate drift between the entire train and test datasets using a model trained to distinguish between them.
+
+    Check fits a new model to distinguish between train and test datasets, called a Domain Classifier.
+    The Domain Classifier is a tabular model, that cannot run on the images themselves. Therefore, the check calculates
+    properties for each image (such as brightness, aspect ratio etc.) and uses them as input features to the Domain
+    Classifier.
+    Once the Domain Classifier is fitted the check calculates the feature importance for the domain classifier
+    model. The result of the check is based on the AUC of the domain classifier model, and the check displays
+    the change in distribution between train and test for the top features according to the
+    calculated feature importance.
+
+    Parameters
+    ----------
+    alternative_image_properties : List[str] , default: None
+        List of alternative image properties names. Must be attributes of the ImageFormatter classes that are passed to
+        train and test's VisionData class. If None, check uses DEFAULT_IMAGE_PROPERTIES.
+    n_top_columns : int , default: 3
```

Comment:
All these parameters should be adjusted. E.g. features / columns -> properties, if that's how we are calling these. Also when talking about categories - categories of the image extracted properties.

Reply:
Yeah I agree, but we need to note we're talking about the importance of general image properties to a domain classifier model (which is not the user model)

Reply:
Added description in docstring